### ISSUE TITLE
Ограниченный пользователь для конкретного отдела.

### ISSUE THREAD

**Issue by Realmagnum** [prog-it/Asterisk-CDR-Viewer-Mod#75]
Хотелось бы такую _странную_ вещь.
Есть к примеру **отдел службы сервиса** и **отдел службы логистики**, в которых по несколько менеджеров.
Начальникам отделов периодически необходимо проверять звонки своих сотрудников и звонки, на которые они ответили.
Хочется **отдельного пользователя**, которому можно прописать номера, которыми будут ограничены все запросы в **CDR**.

**Comment by ramashkaa** [comment]
Мне так-же хотелось-бы некоторое разграничение прав на прослушивание записей для пользователей. Одним можно только эти разговоры слушать, а другим только те.
У меня задача примерно такого содержания:
все видят все CDR-записи,
но прослушивать записи можно только своего филиал (т.е. с компа с IP 192.168.2.х/24 можно прослушивать записи разговоров, которые были с номеров 1001 1002 1003 и на номер 8-495-123-45-67).

**Comment by Asteriskdb** [comment]
Да-да, очень актуальный вопрос. Два совершенно несвязанных раздела не должны видеть записи друг друга. Различаются они контекстом или я могу записывать в accountcode что-нибудь. Например, ip аппарата, чтобы его пользователь мог прослушивать только свои записи.

**Comment by SolarWind13** [comment]
Присоединюсь к просьбе.
Очень не хватает понятия "отдел".
Т.е. возможности задать для пользователя (админа) список номеров со статистикой которых он может работать.
